config/routes.rb

```diff
@@ -49,7 +49,9 @@
 
     resources :advisers, only: [:index, :show, :edit, :update, :destroy]
 
-    resources :firms, only: [:index, :show] do
+    resources :travel_insurance_firms, only: [:index, :show]
+
+    resources :firms, controller: 'retirement_firms', only: [:index, :show] do
```

Comment:
@malcolmbaig that's the only gotcha in this PR. I decided to leave `resources :firms` as it is because I almost endup rewriting a lot of views and renaming them and there was still more and more to be renamed and changed which I kinda feel that it not part of the card of "Firms tab". Please do let me know your point of view 🙏